docs/src/main/asciidoc/spring-cloud-sleuth.adoc

```diff
@@ -319,7 +319,10 @@ spring:
     enabled: false
 zipkin:
   storage:
-    type: mysql
+    type: 
```

Comment:
if you want to hard-code the username and password, the mysql entries need to be at the same level...

```yaml
    mysql:
      host: ${MYSQL_HOST:localhost}
      port: ${MYSQL_TCP_PORT:3306}
      username: ${MYSQL_USER:}
      password: ${MYSQL_PASS:}
      db: ${MYSQL_DB:zipkin}
      max-active: ${MYSQL_MAX_CONNECTIONS:10}
      use-ssl: ${MYSQL_USE_SSL:false}
```